pages/linux/protonvpn-cli-connect.md

```diff
@@ -0,0 +1,28 @@
+# protonvpn-cli connect
+
+> Official client to connect to ProtonVPN from the command-line.
+> More information: <https://github.com/ProtonVPN/linux-cli>.
```

Comment:
```suggestion
> More information: <https://protonvpn.com/>.
```

Reply:
I wasn't sure about the link, but I think the home is not cool because it's hard to find the man page for the command, another option is https://protonvpn.com/support/linux-vpn-setup/

Reply:
Use that, since it still provides `more information` than source code

Reply:
Given this page is in the `linux`, I'd suggest linking to either the existing page or @navarroaxel's suggestion